api/route/router.ts

```diff
@@ -37,8 +38,11 @@ export default function createRouter(namespace: string = '', operations: Operati
     return router;
   }, Router());
 }
-export function useRouter(instance: ExpressInstance, namespace: string = '/', router: Router) {
-  instance.use('/', router);
```

Comment:
may as well drop the 2nd argument too. `namespace doesn't seem to be doing anything anymore.

Reply:
You are right, will fix that